cups/dest.c

```diff
@@ -3400,21 +3413,63 @@ cups_dnssd_unquote(char       *dst,	/* I - Destination buffer */
 }
 
 
+/*
+ * 'cups_gettimeofday()' - Get the sharpest time possible
+ */
+
+static void
+cups_gettimeofday(_cups_time_t *t)
+{
+#if _WIN32
+  t->t = GetTickCount();
+#elif HAVE_CLOCK_MONOTONIC
+  clock_gettime(CLOCK_MONOTONIC, &t->t);
+#elif __sun
```

Comment:
Let's not bother with Solaris-specific functions. We don't have any Solaris testing resources.

Reply:
Backported cupsGetClock() as _cupsGetClock().